utterances.txt

```diff
@@ -83,8 +83,8 @@ UpdateAudio update (audio/music) library
 
 ListenToArtist (play/listen to/shuffle) (/music by/artist) {Artist}
 
-ListenToAlbum (play/listen to) (/album) {Album} by {Artist}
-ListenToAlbum (play/listen to) album {Album}
+ListenToAlbum (play/listen to) (/(/the )album) {Album} by {Artist}
+ListenToAlbum (play/listen to) (/the) album {Album}
```

Comment:
I believe articles are unnecessary, plus this file isn't used by the skill itself.  It's just a framework used by another tool to generate alexa.utterances.